terraform/modules/data-share-service/alarms.tf

```diff
@@ -123,3 +123,18 @@ resource "aws_cloudwatch_metric_alarm" "queue_process_error_number" {
   }
 }
 
+resource "aws_cloudwatch_metric_alarm" "unconsumed_events" {
+  alarm_name          = "${local.alarm_prefix}-growing-unconsumed-events"
+  comparison_operator = "GreaterThanOrEqualToThreshold"
+  evaluation_periods  = "1"
+  threshold           = "500"
```

Comment:
I think the alarm is gonna fire for this much, it may be worth starting very high

Reply:
Sure, this was just the number on the ticket